README.md

```diff
@@ -75,3 +85,46 @@ under `toxic`.
 This is similar to the bias madlibs technique, but has the advantage of using
 more realistic data. One can also use the model's performance on the original
 vs. fuzzed test set as a bias metric.
+
+## Dataset de-biasing
+
+TODO(jetpack): add dataset of additional examples as TSV with columns rev\_id,
+comment, and split. also add tool to recreate dataset from wikipedia dump.
+
+TODO(nthain,jetpack): upload new model trained on new dataset.
+
+This technique mitigates the dataset bias found in the **Dataset bias
+evaluation** section by determining the per-length-bucket deficit of non-toxic
```

Comment:
I suggest reordering for clarity:
"by determining the deficit of non-toxic comments for each of the bias terms split by comment lengths. This deficit is then ameliorated by sampling presumed non-toxic examples from Wikipedia articles."

Reply:
done